src/components/lib/Image.js

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Img from "gatsby-image"
+
+const Image = ({ imageData, className = "" }) => {
```

Comment:
Is this just because we haven't settled on a single image approach yet?

Reply:
Yeah - it's so we can be a little flexible on what we provide as props - whether it's an object that includes and image and alt properties, or just passing in the image itself. Also, using the string is for if we just want to use an svg...in which case we won't use gatsby-image.